scripts/env/dev/init.sh

```diff
@@ -52,14 +52,31 @@ check_supported_kernel() {
 
 #function to install/upgrade cortx-py-utils rpm
 install_cortx_py_utils() {
```

Comment:
please do same changes in other two ansible scripts (release and rpms) just to keep them in sync of dev ansible script

Reply:
fixed.